app/src/main/java/net/gsantner/markor/format/markdown/MarkdownTextModuleActions.java

```diff
@@ -489,4 +509,190 @@ public void onClick(DialogInterface dialog, int id) {
                 });
         builder.show();
     }
+
+    /**
+     * method 'll add text output to current index in editor
+     * as default will be added datetime based on given DT format
+     * optionally 'll be added DT format
+     *
+     */
+    @SuppressWarnings("RedundantCast")
+    private void addDateTime() {
+        android.support.v7.app.AlertDialog.Builder builder = new android.support.v7.app.AlertDialog.Builder(getActivity());
+        final View view = _activity.getLayoutInflater().inflate(R.layout.time_format_dialog, (ViewGroup) null);
```

Comment:
please call it `root`, `viewRoot` or like that, makes it lots easier to work on this :)